com.trollworks.gcs/src/com/trollworks/gcs/weapon/WeaponDamage.java

```diff
@@ -55,12 +59,14 @@
     private double         mFragmentationArmorDivisor;
     private String         mFragmentationType;
     private int            mModifierPerDie;
+    private String         mPercentBonus;
```

Comment:
This should be stored as an integer, not as a string, so that any other calculations that may want to use it in the future don't have to muck with string conversion.

Reply:
Yeah, Done so! 👍